skrub/_data_ops/_estimator.py

```diff
@@ -646,6 +646,12 @@ def cross_validate(learner, environment, *, keep_subsampling=False, **kwargs):
     4    0.85
     Name: test_score, dtype: float64
     """
+    if not hasattr(learner, "__skrub_to_Xy_pipeline__"):
+        raise ValueError(
+            f"`cross_validate` function requires either a Learner object \
+                         or a ParamSearch object, got {type(learner)}."
+        )
```

Comment:
```suggestion
        raise ValueError(
            "`cross_validate` function requires either a Learner object or "
            f"a ParamSearch object, got {type(learner)}."
        )
```

formatting nitpick :)